tensorflow_datasets/core/download/downloader.py

```diff
@@ -172,7 +191,11 @@ def _sync_download(self, url, destination_path):
         opener = urllib.request.build_opener(proxy)
         urllib.request.install_opener(opener)   # pylint: disable=too-many-function-args
       request = urllib.request.Request(url)
-      response = urllib.request.urlopen(request)
+
+      if CA_VERIFY['urllib'] is None:     # disable ssl context check for python version <= 2.7.8
```

Comment:
do I read the code correctly, that what you're doing is just for the ftp:// flow ? 

If yes - please mention it in the CL description

Reply:
Yes. The disabled SSL skipping mechanism is only for ftps:// networks, for python 2.7.8.

Reply:
Cool :)